src/ListBox/ListBox.stories.jsx

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable react/no-multi-comp */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import centered from '@storybook/addon-centered';
+import 'bootstrap/scss/utilities/_screenreaders.scss';
+
+import Button from '../Button';
+import ListBox from '../ListBox';
+import ListBoxOption from '../ListBoxOption';
+
+class ListBoxWrapperForOnSelect extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onSelect = this.onSelect.bind(this);
+
+    this.state = {
+      selectedOption: null,
+      selectedOptionIndex: null,
+    };
+
+    this.fruits = {
+      Apple: '🍎',
+      Orange: '🍊',
+      Strawberry: '🍓',
+      Banana: '🍌',
+    };
+  }
+
+  onSelect(option, index) {
+    this.setState({
+      selectedOption: option,
+      selectedOptionIndex: index,
+    });
+  }
+
+  getSelectedFruitEmoji(fruit) {
+    return fruit ? this.fruits[fruit] : '';
+  }
+
+  render() {
+    const children = Object.keys(this.fruits).map((fruit, index) => (
+      <ListBoxOption
+        key={fruit}
+        onSelect={() => this.onSelect(fruit, index)}
+        style={{ textAlign: 'center' }}
+      >
+        {fruit}
+      </ListBoxOption>
+    ));
+
+    return (
+      <React.Fragment>
+        <span aria-live="polite">
+          Selected Fruit:
+          {this.state.selectedOptionIndex !== undefined ?
+            <span arialabelledby={`list-box-option-${this.state.selectedOptionIndex}`} >{this.getSelectedFruitEmoji(this.state.selectedOption)}</span> :
+            <span className="sr-only">none</span>
+          }
+        </span>
+        <ListBox style={{ width: '200px' }}>
+          {children}
+        </ListBox>
+      </React.Fragment>
+    );
+  }
+}
+
+class ListBoxWrapperForSelectedOptionIndex extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onSelect = this.onSelect.bind(this);
+    this.onButtonClick = this.onButtonClick.bind(this);
+
+    this.state = {
+      reset: true,
+    };
+
+    this.fruits = ['Apple', 'Orange', 'Strawberry', 'Banana'];
```

Comment:
Nit: this could be a little DRY-er if a `fruits` hash/object/dictionary `const` were declared outside of the classes and `this.fruits` were extracted from the keys, e.g. [Object.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys).